analyses/2019_03_agarwalishita_analysis.md

```diff
@@ -0,0 +1,30 @@
+The major difference between the dataset required to train/synthesize ad blocker and data blocker is described in this piece of text.
+
+**Ad blocker:**
+Its data mainly consists of filter rules which comes from EasyList, EasyPrivacy, etc. which consist of known ads on English Language websites all over the world. You can check the whole list [here](https://easylist.to/easylist/easylist.txt).
```

Comment:
This is an incomplete description of the landscape of ad-blockers. If you want to use this as a basis to compare against, I think you need a more complete and nuanced description of the ad-blocking ecosystem seems appropriate and then a justification of your narrowing down to EasyList (which is a block list not an ad blocker).